test/verify/check-machines

```diff
@@ -409,6 +409,12 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.set_input_text("#text-search", "subVmTest2")
         self.waitVmRow("subVmTest2")
         self.waitVmRow("subVmTest1", "system", False)
+        b.set_input_text("#text-search", "")
```

Comment:
Clear the name filter a bit later, so that you can also test that filters combine properly. I. e. 'all running VMs that have "bVmTest1" in the name'